homeassistant/components/light/isy994.py

```diff
@@ -70,7 +70,7 @@ def turn_on(self, brightness=None, **kwargs) -> None:
             _LOGGER.debug('Unable to turn on light.')
 
     @property
-    def state_attributes(self):
+    def device_state_attributes(self):
```

Comment:
TLDR: I don't think the platform needs to report brightness in device state attributes.

The reason why this was added in https://github.com/home-assistant/home-assistant/pull/4152 was that the slider for brightness was missing for this device. But the real reason behind that behavior was actually that `state_attributes` was incorrectly overwritten in the ISYDevicemm, which your PR now fixes. The light component will report brightness attribute in state attributes when the light is on.

Also, now after https://github.com/home-assistant/home-assistant-polymer/pull/168, the brightness slider will always show in the more info panel, even when light is off, as long as the device supports brightness. So the brightness attribute is now longer connected to showing the slider or not.